src/beneuro_data/cli.py

```diff
@@ -678,6 +742,119 @@ def upload_session(
     return True
 
 
+@app.command()
+def up(
+    session_name: Annotated[
+        str, typer.Argument(help="Name of session: M123_2000_02_03_14_15")
+    ],
+    include_behavior: Annotated[
+        bool,
+        typer.Option(
+            "--include-behavior/--ignore-behavior",
+            "-b/-B",
+            help="Download behavioral data (-b) or not (-B)."
+        ),
+    ] = True,
+    include_ephys: Annotated[
+        bool,
+        typer.Option(
+            "--include-ephys/--ignore-ephys",
+            "-e/-E",
+            help="Download ephys data (-e) or not (-E)."
+        ),
+    ] = False,
+    include_videos: Annotated[
+        bool,
+        typer.Option(
+            "--include-videos/--ignore-videos",
+            "-v/-V",
+            help="Download video data (-v) or not (-V)."
+        ),
+    ] = False,
+    include_extra_files: Annotated[
+        bool,
+        typer.Option(
+            "--include-extra-files/--ignore-extra-files",
+            help="Upload extra files that are created by the experimenter or other software.",
+        ),
+    ] = True,
+    rename_videos_first: Annotated[
+        bool,
+        typer.Option(
+            "--rename/--no-rename",
+            help="Rename videos before validating and uploading.",
+        ),
+    ] = True,
+    rename_extra_files_first: Annotated[
+        bool,
+        typer.Option(
+            "--rename-extra/--no-rename-extra",
+            help="Rename extra files (e.g. comment.txt) before validating and uploading.",
+        ),
+    ] = True,
+    processing_level: Annotated[
+        str, typer.Argument(help="Processing level of the session. raw or processed.")
+    ] = "raw",
+):
+    """
+    Upload (raw) experimental data to the remote server.
+
+    Example usage to upload everything of a given session:
+        `bnd up M017_2024_03_12_18_45 -ev`
+    Example to upload the videos and ephys of the last session of a subject:
+        `bnd up M017 -evB`
+    """
+    animal = session_name[:4]
```

Comment:
Can we assume that the subject's name will always be 4 characters? I'd rather not.
How about checking if there is an underscore in the name? If there is, then a session name was passed and the animal name is until the first underscore, otherwise it's an animal name.
That would be consistent with how `data_validation.validate_session_path` checks that the session name starts with the subject's name followed by an underscore.

```suggestion
    session_name = session_or_animal_name if "_" in session_or_animal_name else None
    animal = session_or_animal_name.split("_")[0]
```

Then you can check if `session_name is None` when selecting the function to call.

Reply:
The subject name will always be a letter and 3 numbers, that's not gonna change and is the safest thing to assume.
Can always check for more strict things, but it won't add anything

Reply:
Okay. It might be useful to document this somewhere.